OpenRA.Mods.Common/Widgets/ObserverProductionIconsWidget.cs

```diff
@@ -193,6 +196,20 @@ public override void Draw()
 						Color.White, Color.Black, 1);
 				}
 			}
+
+			var parentWidth = Bounds.X + Bounds.Width;
+			Parent.Bounds.Width = parentWidth;
+
+			var gradient = Parent.Get<GradientColorBlockWidget>("PLAYER_GRADIENT");
+
+			var offset = gradient.Bounds.X - Bounds.X;
+			var gradientWidth = Math.Max(MinWidth - offset, currentItemsByItem.Count * (IconWidth + IconSpacing));
+
+			gradient.Bounds.Width = gradientWidth;
+			var widestChildWidth = Parent.Parent.Children.Max(x => x.Bounds.Width);
+
+			// Add 25 pixels to account for the scrollbar on the left and 25 for gradiend fade-out after the icons.
```

Comment:
Typo: gradiend. Is the comment about the scrollbar still valid?

Reply:
Cant remember, so will remove it.